statsmodels/tsa/statespace/tools.py

```diff
@@ -1863,7 +1863,12 @@ def prepare_trend_spec(trend):
         if trend.ndim > 0:
             polynomial_trend = (trend > 0).astype(int)
         else:
-            raise ValueError('Invalid trend method.')
+            raise ValueError(
+                "Valid trend inputs are 'c' (constant), 't' (linear trend in "
+                "time), 'ct' (both), 'ctt' (both with trend squared) or an "
+                "interable defining a polynomial, e.g., [1, 1, 0, 1] is a + "
```

Comment:
Use single ticks around the \`a + "b*t + ct**3\`. It is hard to read as is

Reply:
Done